src/main/java/uk/gov/hmcts/reform/roleassignment/controller/endpoints/GetAssignmentController.java

```diff
@@ -74,7 +74,8 @@ public GetAssignmentController(@Autowired RetrieveRoleAssignmentOrchestrator ret
         var responseHeaders = new HttpHeaders();
         RoleAssignmentResource body = responseEntity.getBody();
 
-        if (body != null && CollectionUtils.isNotEmpty(body.getRoleAssignmentResponse())) {
+        if (retrieveRoleAssignmentService.isActorCacheFlagEnabled()
```

Comment:
If the flag is not enabled then we should return the eTag with 0 otherwise it could cause a breaking change for the consumer apps.

Reply:
ok, changed as per your comment